en/docs/references/extend/federation/write-a-custom-local-authenticator.md

```diff
@@ -1,24 +1,35 @@
 # Write a Custom Local Authenticator
 
-The default WSO2 Identity Server authenticator is the basic authenticator. It is a local authenticator that authenticates the end users using a connected user store and the provided user name and password. When a user needs to be authenticated with the user store and authorized based on the assigned user role, you can write a custom local authenticator. Let's try out a sample scenario on this.
+A local authenticator in WSO2 identity server authenticates users, who are stored in the local database, using a defined authentication logic.
+
+By default, the local authenticator in WSO2 identity server is the basic authenticator.
+It authenticates end users who are stored in a connected user store using the provided username and password.
+
+WSO2 identity server supports extensibility in local authentication, so that you can implement a different authentication logic by writing a custom local authenticator.
```

Comment:
```suggestion
WSO2 identity server supports extensibility in local authentication so that you can implement a different authentication logic by writing a custom local authenticator.
```